src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { colors } from './colors'
+
+function ensureDepIsInstalled(depName: string) {
+  try {
+    require(depName)
+  } catch (err) {
+    if (err.code === 'MODULE_NOT_FOUND') {
+      console.error(
+        `${colors.red('ERROR:')} ${colors.green(
+          depName
+        )} must be installed as a dependency. Please run \`${colors.green(`npm install ${depName}`)}\`.`
+      )
+      process.exit(1)
+    } else {
+      throw err
+    }
+  }
+}
+
+ensureDepIsInstalled('@nexus/schema')
+ensureDepIsInstalled('graphql')
+ensureDepIsInstalled('@prisma/client')
+
+export * from './plugin'
```

Comment:
Let's make this module export `nexusPrisma`. This should be the function that could be accessed only previously like so:

```ts
import { nexusSchemaPrisma } from 'nexus-plugin-prisma/schema'
```